compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1949,15 +1948,27 @@ private STNode parseParameterRhs(SyntaxKind tokenKind) {
      */
     private STNode parseParameter(SyntaxKind prevParamKind, boolean isParamNameOptional) {
         STNode annots;
+        STNode inclusionSymbol;
         STToken nextToken = peek();
         switch (nextToken.kind) {
             case AT_TOKEN:
                 annots = parseOptionalAnnotations();
+                if (peek().kind == SyntaxKind.ASTERISK_TOKEN) {
+                    inclusionSymbol = parseAsteriskToken();
+                } else {
+                    inclusionSymbol = STNodeFactory.createEmptyNode();
+                }
+                break;
+            case ASTERISK_TOKEN:
+                inclusionSymbol = parseAsteriskToken();
```

Comment:
can use `consume()` and remove `parseAsteriskToken()` method

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/27662